### PR TITLE
Update FAQ #check_in_qr_warning_share

### LINKED ISSUE #3020: PCR test recommended after red tile from check in
## Where to find the issue

In the FAQ https://www.coronawarn.app/en/faq/results/#check_in section

- https://www.coronawarn.app/en/faq/results/#check_in_qr_warning_share "Can this information be voluntarily communicated to the health authority? If so, what information is flowing?"

image

- https://www.coronawarn.app/de/faq/results/#check_in_qr_warning_share "Können diese Informationen freiwillig dem Gesundheitsamt mitgeteilt werden? Wenn ja, welche Informationen fließen?"

## Describe the issue

- This follows on from the issue https://github.com/corona-warn-app/cwa-website/issues/2965

https://www.coronawarn.app/en/faq/results/#check_in_qr_warning_share "Can this information be voluntarily communicated to the health authority? If so, what information is flowing?" says:

"Can this information be voluntarily communicated to the health authority? If so, what information is flowing?

After a red tile warning (high risk) you should quarantine and have a PCR test taken immediately."

A PCR test is no longer the first step after receiving a red tile warning.

- Users are now generally advised to get a rapid antigen test or do a self-test if they see a red tile. If that test returns positive, then they can get a PCR test

National regulations changed on June 30, 2022. See Bundesministerium für Gesundheit. [RKI: Nationale Teststrategie SARS-CoV-2](https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Teststrategie/Nat-Teststrat.html) links to [BMG: Fragen und Antworten zu COVID-19 Tests](https://www.bundesgesundheitsministerium.de/coronavirus/nationale-teststrategie/faq-covid-19-tests.html), including https://www.bundesgesundheitsministerium.de/coronavirus/nationale-teststrategie/faq-covid-19-tests.html#c23925

image

The regulation [TestV](https://www.gesetze-im-internet.de/coronatestv_2021-10/)<br>"Verordnung zum Anspruch auf Testung in Bezug auf einen direkten Erregernachweis des Coronavirus SARS-CoV-2 (Coronavirus-Testverordnung - TestV)" is now changed and extended in time by the [Dritte Verordnung zur Änderung der Coronavirus-Testverordnung Vom 29. Juni 2022](https://www.bundesanzeiger.de/pub/publication/oLkvcL7aBEpNfC03aH7/content/oLkvcL7aBEpNfC03aH7/BAnz%20AT%2029.06.2022%20V1.pdf?inline). 

## Suggested change

Add the following information to the FAQ article https://www.coronawarn.app/en/faq/results/#check_in_qr_warning_share "Can this information be voluntarily communicated to the health authority? If so, what information is flowing?":

"If the Corona-Warn-App indicates a high infection risk with the red card, please get tested with a rapid antigen test or test yourself, preferably several times at intervals of two days. In case a test result is positive, you should confirm it with a PCR test."

### PR DESCRIPTION
**Pull request by larswmh**
Resolves #3020 

[#check_in_qr_warning_share](https://www.coronawarn.app/en/faq/results/#check_in_qr_warning_share)
image
[#check_in_qr_warning_share](https://www.coronawarn.app/de/faq/results/#check_in_qr_warning_share)
image

---
Internal Tracking ID: [EXPOSUREAPP-13583](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13583)

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -1040,7 +1040,7 @@
...
                                 "active": false,
                                 "textblock": [
-                                    "After a red tile warning (high risk) you should quarantine and have a PCR test taken immediately. If the test result is positive, a report is sent to the health authority by the doctor/test center. It is recommended that you share the positive test result via the CWA. In addition, the information can be also be shared manually with the health department using the existing export function of the contact journal."
+                                    "After a red tile warning (high risk) you should get tested with a rapid antigen test or tested by yourself, preferably several times at intervals of two days. In case a test result is positive, you should confirm it with a PCR test. If the test result is positive, a report is sent to the health authority by the doctor/test center. It is recommended that you share the positive test result via the CWA. In addition, the information can be also be shared manually with the health department using the existing export function of the contact journal."
```
Comment (MikeMcC399) [inline src/data/faq.json:1043]: "or tested by yourself" =>
"you should test yourself"
sounds better

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
One small suggestion for the English text (see in-line).

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for reporting issue, suggestions and reviews!

**Review by GisoSchroederSAP — APPROVED** [review]
Acceptable adjustment regarding current recommendations of officials.
Approved.

**Pull request merged**